R/plotCCA.R

```diff
@@ -0,0 +1,540 @@
+#' Plot RDA or CCA object
+#'
+#' \code{plotRDA} and \code{plotCCA} create an RDA/CCA plot starting from the
+#' output of \code{\link[mia:runCCA]{CCA and RDA}} functions, two common methods
+#' for supervised ordination of microbiome data.
+#'
+#' @param object a
+#'   \code{\link[TreeSummarizedExperiment:TreeSummarizedExperiment-constructor]{TreeSummarizedExperiment}}
+#'   or a matrix of weights. The latter is returned as output from \code{\link[mia:runCCA]{calculateRDA}}.
+#' 
+#' @param dimred A string or integer scalar indicating the reduced dimension to
+#'   plot. This is the output of \code{\link[mia:runCCA]{runRDA}} and resides in
+#'   \code{reducedDim(tse, dimred)}.
+#' 
+#' @param ellipse_fill TRUE or FALSE, should ellipses be opacity filled.
+#'   (default: \code{ellipse_fill = TRUE})
+#'
```

Comment:
There is also add_ellipse parameter (which is hidden now) --> You could combine these two parameters --> maybe add_ellipse which can be TRUE, FALSE, "fill" and "color"/"colour". (TRUE could be same as "color". With FALSE, no ellipses are plotted

Reply:
Do you mean that we could control both colour and fill with only one argument (add_ellipse)?

But then ellipses without fill (which look nice) cannot be produced.

Reply:
Ah never mind, now I got it!